test/tax-lot/tax-lot.e2e-spec.ts

```diff
@@ -0,0 +1,92 @@
+import * as request from "supertest";
+import { INestApplication } from "@nestjs/common";
+import { Test } from "@nestjs/testing";
+import { TaxLotModule } from "src/tax-lot/tax-lot.module";
+import { TaxLotRepository } from "src/tax-lot/tax-lot.repository";
+import { getTaxLotGeoJsonByBblQueryResponseSchema } from "src/gen";
+import { TaxLotRepositoryMock } from "./tax-lot.repository.mock";
+import {
+  DataRetrievalException,
+  InvalidRequestParameterException,
+} from "src/exception";
+import { HttpName } from "src/filter";
+
+describe("TaxLots", () => {
+  let app: INestApplication;
+
+  const taxLotRepository = new TaxLotRepositoryMock();
```

Comment:
Instantiate the mock class. Make it a constant that is available throughout the suite